chapters/classes.tex

```diff
@@ -1397,6 +1397,12 @@ \section{Predefined Types and Classes}\label{predefined-types-and-classes}
 The definitions use \lstinline!RealType!\indexinline{RealType}, \lstinline!IntegerType!\indexinline{IntegerType}, \lstinline!BooleanType!\indexinline{BooleanType}, \lstinline!StringType!\indexinline{StringType}, \lstinline!EnumType!\indexinline{EnumType} as mnemonics corresponding to machine representations.
 These are called the \firstuse[primitive type]{primitive types}.
 
+\begin{definition}[Fall-back value]\label{def:fall-back-value}\index{fall-back value}
```

Comment:
Shouldn't it be written without hyphenation?
```suggestion
\begin{definition}[Fallback value]\label{def:fallback-value}\index{fallback value}
```

Reply:
I agree that it should be "fallback".